src/core/Dropdown/Dropdown.test.tsx

```diff
@@ -28,6 +28,9 @@ test(
       'aria-valid-attr-value': {
         enabled: false,
       },
+      region: {
+        enabled: false,
+      },
```

Comment:
These need further investigation, an additional ticket needs to be created and libs axe utility needs to be updated to do a deep merge for options provided as parameter and util defaults.

Reply:
There is now a shallow merge of options.rules with default options. In case same key is provided with options.rules, it overrides corresponding default option rule. Deep merge is not desirable for rules object since it could result into unwanted rule-sets that are difficult to understand.

Reply:
Created an issue to our backlog to investigate if we should enable region rule.